docs/contributing/development.md

```diff
@@ -1,62 +1,54 @@
-# Development environment setup
+# Development
+
+## Getting the code
 
 1. Fork [nsidc/earthaccess](https://github.com/nsidc/earthaccess)
 1. Clone your fork (`git clone git@github.com:{my-username}/earthaccess`)
 
-`earthaccess` uses Poetry to build and publish the package to PyPI, the defacto Python repository. In order to develop new features or fix bugs etc. we need to set up a virtual environment and install the library locally. We can accomplish this with Conda and Poetry, or just with Poetry.  Both workflows achieve the same result.
+In order to develop new features or fix bugs etc. we need to set up a virtual
+environment and install the library locally.
 
-### Using Conda
+## Quickstart development
 
-If we have `mamba` (or `conda`) installed, we can use the environment file included in the `ci` folder. This will install all the libraries we need (including Poetry) to start developing `earthaccess`:
+The fastest way to start with development is to use nox. If you don't have nox,
+you can use `pipx run nox` to run it without installing, or `pipx install nox`.
+If you don't have pipx (pip for applications), then you can install with
+`pip install pipx` (the only case were installing an application with regular
+pip is reasonable). If you use macOS, then pipx and nox are both in brew, use
+`brew install pipx nox`.
 
-```bash
-mamba env update -f ci/environment-dev.yml
-mamba activate earthaccess-dev
-poetry install
+To use, run `nox`. This will typecheck and test using every installed version of
+Python on your system, skipping ones that are not installed. You can also run
+specific jobs:
+
+```console
+$ nox -s typecheck              # Typecheck only
+$ nox -s tests                  # Python tests
+$ nox -s build_docs -- --serve  # Build and serve the docs
+$ nox -s build_pkg              # Make an SDist and wheel
 ```
 
-After activating our environment and installing the library with Poetry we can run Jupyter lab and start testing the local distribution or we can use `make` to run the tests and lint the code.
-Now we can create a feature branch and push those changes to our fork!
+Nox handles everything for you, including setting up a temporary virtual
+environment for each run.
 
-### Using Poetry
+## Manual development environment setup
 
-If we want to use Poetry, first we need to [install it](https://python-poetry.org/docs/#installation). After installing Poetry we can use the same workflow we used for Conda, first we install the library locally:
+Create and activate a virtual environment with `venv`, which comes by default with
+Python, in the `.venv` directory:
 
 ```bash
-poetry install
+python -m venv .venv
+source .venv/bin/activate
```

Comment:
Any reason this isn't exposed as a `nox` command?

Seems like pushing as much into `nox` as possible would further simplify things, and even make it easier to change things in the future (if desired) without requiring a contributor to do anything differently (i.e., I see "nox" as the interface and the stuff in the noxfile as the "implementation", such that "nox" encapsulates things, allowing use to change the implementation as necessary, without affecting the nox user).

Reply:
I haven't done or seen that done before -- I assumed that environment activation would not be possible to do within a `nox` command because the command doesn't execute in the current shell?

Reply:
Sorry, yes, I was looking only at the venv creation. Activation, of course, must be done explicitly by the user, so that's one bit that "leaks" through to the user, which cannot be encapsulated/hidden by `nox` (or `make`).

I'm suggesting we put as much as possible behind `nox`, so that as much as possible, the user uses `nox` as the one way of doing things.

Reply:
I don't know if I agree that we should position `nox` as the "one way" of doing things. I definitely think it's valuable as a quickstart to common tasks, though. E.g.: "I don't know how to install a dev environment but I would be embarassed if I pushed my simple error message change and it didn't pass tests"